libmate-desktop/mate-bg.c

```diff
@@ -1163,6 +1163,7 @@ mate_bg_create_surface_scale (MateBG      *bg,
 
 	cairo_surface_t *surface;
 	cairo_t *cr;
+    GdkDisplay *display;
```

Comment:
Indent

Reply:
Needs to be fixed....

Reply:
Btw. here a tab is needed.

Reply:
Found and fixed, BTW that Pluma setting to show tabs and spaces is a real game changer for code!